### DISCUSSION TITLE
Treating Barotrauma: medical overhaul idea

### DISCUSSION THREAD

**Discussion by mLuby** [FakeFishGames/Barotrauma#5220] [Ideas and suggestions]
# ![bandages](https://static.wikia.nocookie.net/subsurface_gamepedia/images/0/0c/Bandage.png/revision/latest/scale-to-width-down/40?cb=20200113233109) Treating Barotrauma: medical overhaul idea ![saline](https://static.wikia.nocookie.net/subsurface_gamepedia/images/e/e3/Saline.png/revision/latest/scale-to-width-down/40?cb=20200113233141)

_Hi all, I have some long-form thoughts on the current medical system and where it could go._

_I'll start by saying that **I appreciate Barotrauma**'s choice to eschew the ubiquitous, simplistic "hit points" system and instead create a realism-oriented system with various afflictions, diagnoses, methods of treatment, and medical items with side effects._

**TL:DR**:
- High character **medical stats** should make medicines heal more per use, decreasing treatment cost by saving items as well as reducing the side effects of each use.
- High **player skill** in **medical mini-games** should mean quicker diagnosis and treatment as well as reducing treatment side effects (e.g. from a shaky scalpel, missing the vein, weak sutures, etc).
- Add a consumable **Broad-Spectrum Stabilizer item and station doctor** to provide an expensive alternative to having a doctor as part of your crew.

_As I'll explain, these changes will decrease new player difficulties with the medical system, increase the value of experienced players in the crew, and encourage keeping characters alive—what medicine is for, after all._

## Problem: gating by role, stats, knowledge, skill, wiki, items

The end goal of someone using Barotrauma's medical system is to heal one or more afflictions, thereby restoring the patient's vitality and preventing them from dying. The question is: what should the player have to do to accomplish that goal? Most games make it a simple 1-click affair: drink a health potion or cast a healing spell. But in Barotrauma, there isn't a one-medicine-heals-all item _\*cough\* morphine \*cough\*_; player knowledge of which items heal which afflictions is what determines whether they can accomplish their healing goal. Here are different ways of determining whether and how effectively the player can heal:

**Player Knowledge**: you know "sore throat" means Husk Infection, now you have a diagnosis. You know Husk Infection needs Calyxanide, so you use that to fix the problem. Pros: rewards players learning; cons: is wiki-gating for new players, any role can diagnose and treat.

**Wiki Access**: you look up some complex symptoms O2=84.2+BMP=73-89 on the wiki table and it says "husk" so now you know what to treat. Pros: prevents simple memorization (see player knowledge-based). Cons: breaks immersion, contrived, any role can diagnose.

**Item Stockpile**: if you have medical goggles or use "Medical Nanites" you can heal with the right medicines. Pros: groups without doctor can still heal, albeit more limited/expensively, uses some player/wiki knowledge. Cons: ignores player skill, any role can diagnose. **This is mostly what Barotrauma's medical system is now.**

**Character Role**: only Doctor role characters can heal. Pros: new players not limited. Cons: all crew need a doctor, ignores non-doctor player knowledge and skill.

**Character Stats**: if your character has > 40 medical stats you can heal. Pros: new players not limited. Cons: most crews still need a doctor, ignores player knowledge and skill.

**Player Skill**: medical mini-games allow players to use skill to get a better medical outcomes. Pros: rewards player skill, can lead to moments of high tension. Cons: may feel repetitive. Any role can diagnose and treat.

_I propose the following combination of these (except wiki-gating, since it breaks immersion)._

## Solution: stabilizers, skill-based treatment, stat-based healing

1. Anyone can **self-administer an expensive, idiot-proof stabilizer** to keep themselves alive long enough to get to a station so an **expensive station doctor can heal them**, but the **player is impaired until then** (e.g. slower movement, debuffed stats). Combat-oriented characters should spawn in with one of these instead of Steroids.

2. If a player knows medical but isn't a Doctor character that round, they can still provide limited aid to their party's recovery efforts by **playing medical mini-games to diagnose and treat afflictions**, thus skipping the expensive stabilizers to directly use side-effectful treatments. Not as good as having a dedicated doctor but still noticeably rewards the player's knowledge and skill. Note that doing badly at a mini-game is expected—**poor mini-game play causes negative side effects** like blood loss, minor wounds, infection. So novice or clumsy or rushed players can still heal but it's better to be under the knife of a calm, steady-handed professional.

3. Characters with **high medical stats apply medicines more effectively**. So since each treatment heals more, they should save a lot of money by consuming fewer expensive medicines, and the side effects caused by each medicine use are lessened as well. Additionally, cap non-doctor characters' medical stats at 70, so only doctors can get medical stats all the way to 100. _For balancing, I'd suggest

## Scenarios
Let's imagine Patty gets seriously injured half-way through a salvage mission.

- _Patty is on her own._ Patty takes an expensive broad-spectrum stabilizer and for the rest of the mission she moves slower and isn't as good at her normal tasks. At the next station, Patty pays a doctor to fix her up, and buys another expensive broad-spectrum stabilizer, just in case.

- _Ned, who's relatively new to Barotrauma and playing as a high medical stat Doctor, treats Patty._ Ned is still bad at the medical mini-games, but his character's medical stats are good so his treatments are effective. Ned makes a bunch of mistakes during the mini-games so the treatment takes a while and Patty receives a lot of minor injuries from his botched efforts. But Ned didn't have to use many items for the treatment so Patty has fewer side effects from those than she might otherwise, and the treatment wasn't too expensive in terms of items consumed. Patty is stable and able to return to her duties for the rest of the mission, though over a number of missions she accumulates more and more of these injuries from Ned's botched attempts until eventually those injuries themselves require treatment.

- _Eve, who's experienced at Barotrauma and this medical system and playing a low medical stat Captain, treats Patty._ Eve is excellent at the medical mini-games, so she does't make mistakes and Patty escapes further injury. However Eve's character needs to use more items to fully heal Patty, which is somewhat costly, and those extra doses cause side effects for Patty. Patty is stable and able to return to her duties for the rest of the mission, though she may require treatment for those side effects, especially if they accumulate over a number of missions.

|Player Minigame Skill|Medical Stats|Outcome|
|--|--|--|
|High|High|Healed with minimal injuries, drug side effects, and items consumed.|
|High|Low |Healed with minimal injuries but more items are used and there are more side effects.|
|Low |High|Healed despite a number of minor injuries, but minimal drug side effects and items consumed.|
|Low |Low |Healed despite a number of minor injuries and significant drug side effects from having to use many items.|

## Medical Mini-games
Access these by equipping the required items (e.g. Stethoscope, Scalpel, Needle, etc.) and opening the patient's health screen.

- **Diagnosis**: check for pulse/breathing, use Stethoscope around the body to "listen" for cardiovascular problems. Feel around for internal damage, bleeding, and bone breaks. Hit patient's joints with a toy hammer to check for nerve damage? _You don't need to diagnose before treating, but you might miss hidden symptoms or injuries._
- **Surgery**: use Scalpel to make incision then extract/implant/align something. Nearly always followed by stitching up the wound.
- **Suture**: use Needle and Organic Fiber to stitch up an incision or wound.
- **Inject**: use Bandage as tourniquet to increase vein visibility over time, stick Needle in vein, inject the medicine, remove Needle.
- **Topical**: use Antibiotics, Salves, and Bandages to clean and treat the flesh.
- **CPR**: its effectiveness is determined by where you press, how "hard" you press (click duration), and how long between presses. Cannot be performed through a diving suit. Mods could add Defibrillator and Intubation Pump to assist.

## Afflictions

|  |Affliction|Treatment(s)|
|--|--|--|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/f/f5/Affliction_Oxygen_Low.png/100px-Affliction_Oxygen_Low.png">|Hypoxia|CPR or Surgery(implant tracheotomy pipe)+Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/e/e8/Affliction_Husk_Infection.png/100px-Affliction_Husk_Infection.png">|Parasite|Surgery(extract parasite)+Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/c/c1/Affliction_Bloodloss.png/100px-Affliction_Bloodloss.png">|Internal Bleeding|Surgery(align artery)+Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/c/c2/Affliction_Burn.png/100px-Affliction_Burn.png">|Burns|Topical(ointments, bandages)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/4/49/Affliction_Blunt_Force_Trauma.png/60px-Affliction_Blunt_Force_Trauma.png">|Broken Bones|Surgery(align bones)+Suture+Topical(Splint)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/8/8c/Affliction_Organ_Damage.png/60px-Affliction_Organ_Damage.png">|Shrapnel|Surgery(extract shrapnel/object)+Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/d/dd/Affliction_Bleeding.png/100px-Affliction_Bleeding.png"><img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/2/28/Affliction_Lacerations.png/60px-Affliction_Lacerations.png"><img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/8/8b/Affliction_Bite_Wounds.png/60px-Affliction_Bite_Wounds.png"><img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/4/44/Affliction_Gunshot_Wound.png/60px-Affliction_Gunshot_Wound.png">|Open Wound|Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/4/4f/Affliction_Radiation_Sickness.png/100px-Affliction_Radiation_Sickness.png">|Radiation|Inject(bone marrow) or Surgery(extract tumor)+Suture|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/c/c7/Affliction_Space_Herpes.png/100px-Affliction_Space_Herpes.png">|Infection|Inject(broad spectrum antibiotics)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/7/72/Affliction_Stun.png/100px-Affliction_Stun.png">|Paralysis|Inject(anti-paralysis)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/f/f6/Affliction_Psychosis.png/100px-Affliction_Psychosis.png">|Hallucinations|Inject(anti-psychotic)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/c/c8/Affliction_Nausea.png/100px-Affliction_Nausea.png">|Poisons|Inject(antidote)|
|<img width="30" src="https://barotraumagame.com/baro-wiki/images/thumb/9/9b/Affliction_Internal_Damage.png/100px-Affliction_Internal_Damage.png">|Debilitating Pain|Inject(painkiller)|

**Infection** can happen when an injury isn't cleaned, so don't forget to include Topical(antibiotics) amid other treatment steps.

### _That's it! [Do you concur?](https://www.youtube.com/watch?v=UnJytdCvhMI)_

P.S. Hospital Bed should stabilize any character dragged to it.

P.S. I'd prefer if the medical goggles (AKA Health Scanner HUD) were removed and replaced by a medical _machine_ like an Auto-Doc that acts as a diegetic means of telling players new to doctoring what they should do next.

P.S. Consider adding 2 afflictions: _Circulatory Shock_=extremely low blood pressure, which often causes _Cardiac Arrest_=heart stops due to lack of blood, lack of oxygen, or neuro-electrical disruption.

**Comment by Veritius** [comment]
Figuring out all of this in a combat situation, especially in a campaign where character death is permanent, would be a nightmare and probably not that fun.
And the minigames... don't really fit Barotrauma. Like, at all.

**Comment by ubertpendragon** [comment]
The current medical system is in a somewhat strange spot, so here's a comprehensive list of good/meh/bad

**The good**
- Afflictions are handled well, with decent amounts of variety, and will probably be expanded further
- Medicines having side-effects is a nice touch and helps deepen the healing process with unwanted interactions

**The meh**
- Medical skill serves little function and needs to have more impact on healing, but it's hard to strike a good balance so non-medics could still apply first aid.
- Morphine is the end-all cure-all. It kind of defeats the point of having a complex system, but at the same time having too many meds for too many afflictions could be problematic.
- Match-type medicine (item X cures disease X) is straightforward, but the more afflictions there are, the more cumbersome it gets

**The bad**
- The UI lies with "suggested" treatments, confusing new players. Other UI elements are useless, such as BPM, or even limbs (see below).
- Damage localization is for the most part useless, only serving a flavor purpose, since all damage is actually somewhat spread across the body. And there isn't much to improve on, since at most you could disable arms or legs for broken bones

So that's my diagnosis anyway. As for possible improvements?

**Low medical skill should matter more**. Hiding afflictions by making them "Unknown" **(?)**, so non-medics would have to make an educated guess. Even more fun if the patient is unconscious and in the hands of unqualified personel. Also, increasing the amount of existing side effects on low med skill _(say, a consequence of poor dosage)_ 

**Adding stabilizers as universal treatment** rather than morphine is a good idea, with the possibility of definitively curing afflictions in outposts or using specific medicine you can craft on the fly. It'd also mean low medical skill can still safely use them instead

Combining both ideas, **adding a diagnosis phase?** With the idea of "unknown" afflictions, separating _causes_ and _symptoms_. Open wound causes blood loss. Blood loss is the SYMPTOM, Open wound is the CAUSE; you can treat the symptom but as long as the cause isn't treated it will keep creating symptoms: Treating symptoms is "stabilizing". Radiation could cause Burns, Organ damage could cause blood loss... but coming up with a whole diagnosis stage to turn "unknown" into "known" is the hard part. Lying on a medical bed for example, or watching how side-effects evolve... lots of things to think about


Minigames is its own can of worms; the novelty can wear off quickly. If engineers and mechanics don't have repair minigames, I doubt medical will have as well, but who knows.

Here's a medical rework just in case:
![Baromed2](https://user-images.githubusercontent.com/73229309/111210841-047d2680-85ce-11eb-9060-bfba6d512e03.png)

**Comment by Krzeszny** [comment]
Small feedback: IRL bandages don't heal burns; they prevent burns from healing. Plastiseal _could_ work but antibiotic glue and other ointments would be best-suited for burns.
Also, there's only internal bleeding in this suggestion, so what about non-internal bleeding? 

Bandages should either slow down or treat all kinds of non-internal bleeding, including bite wounds, tissue injuries, and lacerations (the last 2 would still require stitching), and stop bleeding from gunshot wounds, but bullets also cause internal injuries which require surgery. Internal injuries should worsen when not treated.

Speaking of non-internal bleeding... makes me wonder why lacerations and bite wounds are even classified as internal injuries. That's nonsense.

**Comment by Udrakan** [comment]
Plastiseal for mild long term healing of bluntforcetrauma. Elastic bandages like this are used for this kind of stuff irl

**Comment by CapoFantasma97** [comment]
Hit points are more like a "consciousness level" in this game. When they reach zero you don't die, you go unconscious. The gravity of your conditions will kill you (excessive O2 deprivation, blood loss, internal damage, extensive burns, etc).

Therefore, I don't see it as simplicistic, but rather well thought and interesting.

Instead, I'd like to see a better variety in treatment. Currently, most conditions can be cured just by 1 item: morphine. Despite being merely a painkiller drug, it heals all kinds of traumas and internal damages, burns, and more. Bandages are very situational (only when bleeding out) and saline/blood pack come into play when the blood loss is excessive and you want someone conscious faster, or healthy.

There's also specific poison related antidotes, but those are very situational and rare to be used.

There is a bunch of medicines that currently only vary in potency: it would be cool to have a horizontal variety instead of vertical, where treating certain conditions could be done only by certain drugs. Pomegrenade and antibiotics glue already cure burns, why should morphine be able to do that as well, for example?

**Reply by Krzeszny** [reply]
> Pomegrenade and antibiotics glue already cure burns, why should morphine be able to do that as well, for example?

Because healing burns with 6 bandages or 6 tubes of antibiotic glue wastes resources. 

Making antibiotic glue heal burns everywhere would help, as would reverting some medicine to multiple-use. I have nothing against partials in stacks. Partials in stacks already work correctly - you take the partial away from the stack and the rest is full. The only problem is that you can't put partials back to a stack, which could be easily solved.

**Reply by CapoFantasma97** [reply]
> 
> 
> > Pomegrenade and antibiotics glue already cure burns, why should morphine be able to do that as well, for example?
> 
> Because healing burns with 6 bandages or 6 tubes of antibiotic glue wastes resources.

Which is a matter of busted balance rather than resources. Exactly no one would use them, because morphine does everything and it's far more effective.
It's just a matter of changing 1 number
